game/scripts/npc/abilities/troll_warlord_fervor.txt

```diff
@@ -21,12 +21,12 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "attack_speed"                                    "15 20 25 30 35 42"
+        "attack_speed"                                    "5 10 15 20 25 30" //OAA
```

Comment:
Might be a bit too harsh of a nerf. maybe something more like 8/12/16/20/25/30?

Reply:
It will be a little rough in early game, but its basically the same hero when he gets some items.

Reply:
at level 1 it's 180 attack speed max to 50....
that's kinda feelsbadman